components/dashboard/src/teams/TeamBilling.tsx

```diff
@@ -151,163 +151,180 @@ export default function TeamBilling() {
+                                            Members
+                                        </div>
+                                        <div className="font-semibold text-base text-gray-600 dark:text-gray-400">
+                                            {members.length}
+                                        </div>
+                                        <div className="mt-4 font-medium text-base text-gray-400 dark:text-gray-600">
+                                            Next invoice on
+                                        </div>
+                                        <div className="font-semibold text-base text-gray-600 dark:text-gray-400">
+                                            {guessNextInvoiceDate(teamSubscription.startDate).toDateString()}
+                                        </div>
+                                        <div className="flex-grow flex flex-col items-stretch justify-end">
+                                            <button
+                                                onClick={() => {
+                                                    if (team) {
+                                                        ChargebeeClient.getOrCreate(team.id).then((chargebeeClient) =>
+                                                            chargebeeClient.openPortal(),
+                                                        );
+                                                    }
+                                                }}
+                                                className="m-0"
+                                            >
+                                                Manage Billing or Cancel
+                                            </button>
+                                        </div>
                                     </div>
-                                </div>
-                            </SolidCard>
-                        )}
-                    </>
-                )}
-            </div>
-            <div className="mt-4 text-gray-500">
-                Team Billing automatically adds all members to the plan.{" "}
-                <a href="https://www.gitpod.io/docs/team-billing" rel="noopener" className="gp-link">
-                    Learn more
-                </a>
-            </div>
+                                </SolidCard>
+                            )}
+                        </>
+                    )}
+                </div>
+                <div className="mt-4 text-gray-500">
+                    Team Billing automatically adds all members to the plan.{" "}
+                    <a href="https://www.gitpod.io/docs/team-billing" rel="noopener" className="gp-link">
+                        Learn more
+                    </a>
+                </div>
+            </>
+        );
+    }
+
+    const showUBP = BillingMode.showUsageBasedBilling(userBillingMode);
```

Comment:
In fact, why is this decision made on the user's billing mode? 🤔 Shouldn't it instead be made on the team billing mode?

Reply:
This :dart: Fixed!